tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature

```diff
@@ -227,64 +227,67 @@ Feature: Share by public link
     When the public uses the webUI to access the last public link created by user "user1" with password "pass123"
     Then file "lorem.txt" should be listed on the webUI
 
-  @skip @yetToImplement
   Scenario: public should not be able to access a public link with wrong password
-    When the user creates a new public link for folder "simple-folder" using the webUI with
-      | password | pass123 |
-    And the public tries to access the last created public link with wrong password "pass12" using the webUI
+    Given user "user1" has shared folder "simple-folder" with link with "read" permissions and password "pass123"
+    When the public uses the webUI to access the last public link created by user "user1" with password "pass12"
     Then the public should not get access to the publicly shared file
 
-  @skip @yetToImplement
   Scenario: user shares a public link with folder longer than 64 chars and shorter link name
-    Given user "user1" has moved folder "simple-folder" to "aquickbrownfoxjumpsoveraverylazydogaquickbrownfoxjumpsoveralazydog"
-    And the user has reloaded the current page of the webUI
+    Given user "user1" has renamed folder "simple-folder" to "aquickbrownfoxjumpsoveraverylazydogaquickbrownfoxjumpsoveralazydog"
+    And user "user1" has logged in using the webUI
     When the user creates a new public link for folder "aquickbrownfoxjumpsoveraverylazydogaquickbrownfoxjumpsoveralazydog" using the webUI with
       | name | short_linkname |
-    And the public accesses the last created public link using the webUI
+    When the public uses the webUI to access the last public link created by user "user1"
```

Comment:
And the public uses the webUI to access the last public link created by user "user1"

Reply:
:rocket: